Source/JavaScriptCore/Scripts/copy-profiling-data.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+if [ -z "${PROFILE_DATA_FLAGS}" ]; then
+    exit 0;
+fi
+
+src="${SCRIPT_INPUT_FILE_0}"
```

Comment:
I feel like longer and more descriptive names would be more in line with WebKit coding style.